src/toil/provisioners/azure/create-azure-vm.yml

```diff
@@ -0,0 +1,125 @@
+# Copyright (C) 2018 UCSC Computational Genomics Lab
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Launch Azure VM
+  hosts: localhost
+  connection: local
+  gather_facts: False
+  vars:
+    dnsname: '{{ vmname }}.westus.cloudapp.azure.com'
+    # The DNS name might not be right depending on your region!!
+    ip: "{{ lookup ('dig', '{{ dnsname }}') }}"
+
+  tasks:
+  - debug: msg="Public DNS name {{ dnsname }} resolved to IP {{ ip }}."
+# The following check is disabled so that the playbook does not stop upon existence of DNS name. Otherwise
```

Comment:
Are we going to leave this check disabled? If so, we should remove this (and line 22) so we can remove the `dnspython` dependency altogether.

Reply:
Is this just checking if a machine with that name exists already? I don't think that is needed.